weblab/entities/views.py

```diff
@@ -1013,3 +1013,43 @@ def get(self, request, *args, **kwargs):
         return JsonResponse({
             task: result
         })
+
+
+class EntityRunExperimentView(LoginRequiredMixin, EntityTypeMixin, DetailView):
```

Comment:
We probably also want this to inherit from `PermissionRequiredMixin` checking if the user can run experiments:
```python
    permission_required = 'experiments.create_experiment'
```

Reply:
Done now fixing tests !!